compiler/passes/initializerRules.cpp

```diff
@@ -2318,3 +2328,61 @@ FnSymbol* buildClassAllocator(FnSymbol* initMethod) {
 
   return fn;
 }
+
+static void buildTypeFunction(FnSymbol* fn) {
```

Comment:
Perhaps choose a name that is a little more descriptive than "fn" and that helps
to remind the reader that this is constructing a type function based on an initializer?